### DISCUSSION TITLE
Spell Checking in markdown limited to english?

### DISCUSSION THREAD

**Discussion by ArturRuta** [requarks/wiki#7359] [Help / Questions] [answered]
Working with wiki.js almost exclusively in the markdown editor.
Have been missing spell checking. 
As far as I can see in the articles arround spell checking uses the browser capabilities in the visual editor and the editor capabilities in the markdown editor (suposedly a bit more limited)
But I'm not seeing any kind of spell checking in the editor.  No highlighting, no button for checking, nothing.
May this be due to the fact that the languaje I've configured is not english?
Am I missing something?

Thanks a lot in advance.

By the way, I'm having a great time moving my technical documentation to wiki.js. Thank you all the contributors.

**Comment by NGPixel** [comment, marked as answer]
You need to click the "Toggle Spellcheck" button in the top-right corner of the editor when the preview pane is opened. Note that this is using your browser built-in spellchecker.

**Comment by ArturRuta** [comment]
Completely missed it !!  Thank you

The behavior is a bit strange however. As soon as I type anything in the markdown window...spell checking gets deactivated in the preview. So, if I have to correct several errors it becames a bit tedious, switching it on again. Is this behavior to be expected? (I may be since we in fact have leaved the "regular" html pane)

Thanks again.

**Reply by NGPixel** [reply]
Yes, this is a limitation of how the spellchecker is used.

**Comment by ArturRuta** [comment]
Will live with this for the time being. Not a critical issue for my use case altough it may be for others.

Out of curiosity... Any evolution planned in this area? Have found no reference in the published materials concerning 3.0 and have not tried the alpha
Just curiotisy, i fully understant no commitments can be made. 3.0 is hugue on itself. 

After this I'll be clossing the discussion and thanks again for such beautiful tool.
